c++/src/kj/async.c++

```diff
@@ -1460,6 +1463,7 @@ FiberStack::FiberStack(size_t stackSizeParam)
   if (_setjmp(impl->originalJmpBuf) == 0) {
     setcontext(&context);
   }
+#endif
 #else
   KJ_UNIMPLEMENTED(
```

Comment:
Can we bring back the custom error message for KJ_NO_EXCEPTIONS:

```
#if KJ_NO_EXCEPTIONS
    KJ_UNIMPLEMENTED("Fibers are not implemented because exceptions are disabled");
#else
    KJ_UNIMPLEMENTED(
      "Fibers are not implemented on this platform because its C library lacks setcontext() "
      "and friends. If you'd like to see fiber support added, file a bug to let us know. "
      "We can likely make it happen using assembly, but didn't want to try unless it was "
      "actually needed.");
#endif
```

Reply:
Done. I am wondering if it's d3adc0d3() now with an #error statement in the async.h?

Reply:
No... that error will only happen if `KJ_USE_FIBERS=1`, but this code here is for `KJ_USE_FIBERS=0`.